packages/wrangler/src/d1/migrations/apply.tsx

```diff
@@ -37,7 +33,7 @@ export function ApplyOptions(yargs: CommonYargsArgv) {
 	return MigrationOptions(yargs).option("batch-size", {
 		describe: "Number of queries to send in a single batch",
 		type: "number",
-		default: DEFAULT_BATCH_SIZE,
+		deprecated: true,
```

Comment:
Even if this is being deprecated, I think we should keep the default value or else it'd be a breaking change in behaviour

Reply:
Unless this option isn't being used at all anymore, in which case it's a breaking change no matter what?

Reply:
The old version couldn't handle too much SQL in one call, so chunked it up using this `batch-size`. The new one does it all in one go, so there's no batching.

So yes, it's no longer used at all, but I don't think that constitutes it being a breaking change. The commands still do exactly what they did previously, just the `batch-size` option has no effect any more, so I deprecated it.